checker/definitions/simple.d.ts

```diff
@@ -22,6 +22,15 @@ declare function compile_type_to_object<T>(): any performs const compile_type_to
 
 // declare var undefined: undefined;
 
+interface ImportEnv {
+  [key: string]: string | undefined;
+}
+
+interface ImportMeta {
+  @Constant
```

Comment:
The `@Constant` can be removed, it is not constant in the sense of `readonly` but instead for putting on methods that can be computed at compile time in a custom way. I need to change its name as it is confusing 😅. Also I think `static` needs to be removed as it isn't allowed in inferfaces.

Reply:
The same for `TypeId`, I was just experimenting (`Copy-Pasted` from other implementations) and that works. The name is perfect.